spack_cloud_env/.spack-env/view

```diff
@@ -0,0 +1 @@
+/Users/moha907/projects/exago-cloud/exago/spack_cloud_env/.spack-env/._view/hduabxzdnj2e7ab2cd3vs7niwmp3te3g
```

Comment:
Lets make sure to gitignore any spack environment files and remove them from this PR